person.rb

```diff
@@ -1,33 +1,36 @@
-require_relative './nameable'
-
-class Person
-  def initialize(age, name = 'Unknown', parent_permission: true)
-    self.age = age
-    self.name = name
+require './nameable'
+require 'date'
+require './rental'
+
```

Comment:
- Kindly In order to include classes, modules, etc in other files you have to use require_relative or require, and when it is some dependency on locating in gem kindly use require `ex: require 'date'.` so here above  I suggest you use require_relative instead of require for nameable `eg: require_relative  './nameable'`  and for  date file use require  ex: require 'date' it will make your code looks more professional   :+1: :+1: :+1: